packages/excalidraw/locales/en.json

```diff
@@ -238,7 +238,8 @@
     "resetLibrary": "This will clear your library. Are you sure?",
     "removeItemsFromsLibrary": "Delete {{count}} item(s) from library?",
     "invalidEncryptionKey": "Encryption key must be of 22 characters. Live collaboration is disabled.",
-    "collabOfflineWarning": "No internet connection available.\nYour changes will not be saved!"
+    "collabOfflineWarning": "No internet connection available.\nYour changes will not be saved!",
+    "localStorageOverflow": "Save failed. Too many items!"
```

Comment:
```suggestion
    "localStorageOverflow": "Couldn't save the scene locally. Too many elements!"
```

Reply:
I adjusted the message, though longer strings do change the aesthetics. Here's a comparison:
![Screenshot 2024-12-11 at 9 34 00 AM](https://github.com/user-attachments/assets/6d74dfd1-8a2e-4356-ac2f-77b2bae68a2b)

I implemented version 3, which uses `text-wrap: "balance"`

Versions 1, 2, and 3 are all mobile compatible. Version 4 will require media queries to revert to the original toast size for smaller screens, which I can implement if preferred.